test/controllers/dashboard_controller_test.exs

```diff
@@ -283,8 +283,8 @@ defmodule HexWeb.DashboardControllerTest do
     assert redirected_to(conn) == "/dashboard/email"
     assert get_flash(conn, :info) =~ "verification email has been sent"
 
-    {subject, contents} = HexWeb.Mail.Local.read("new@mail.com")
-    assert subject =~ "Hex.pm"
-    assert contents =~ "email/verify?username=#{c.user.username}"
+    [email] = Bamboo.SentEmail.all
```

Comment:
@fteem build is failing on this line - could you double-check please? I believe this old test was wrong and was passing for the wrong reason, because our previous test mailer implementation didn't handle well multiple emails to the same user :)